components/raftstore/src/store/peer.rs

```diff
@@ -5273,10 +5279,40 @@ where
         } else {
             RegionReplicationState::IntegrityOverLabel
         };
+        // Once the DR replicas catch up the log during the `SyncRecover` phase, we can
+        // enable group commit.
+        if state == RegionReplicationState::IntegrityOverLabel
+            && self.dr_auto_sync_state == DrAutoSyncState::SyncRecover
+        {
+            self.raft_group.raft.enable_group_commit(true)
```

Comment:
I notice in `switch_replication_mode` it would call `calculate_commit_group` and assign commit group if group commit enabled. But here doesn't do it.

Reply:
yes, the `calculate_commit_group` calculate according to `GlobalReplicationState` change, I think it's only to calculate at that time, here we do need to recalculate.

Reply:
updated.